docs/modules/ROOT/pages/general-info/technical-preview.adoc

```diff
@@ -1,11 +1,11 @@
 [[overview]]
 = Features in Technical Preview
 
-Some features that are new in the latest version of Payara Server or Payara Micro are released in _technical preview_. This means that the feature is ready to be used and also commercially supported, but we recommend trying out the feature extensively in a test environment before using in production. 
+Some features that are new in the latest version of Payara Server Community or Payara Micro Community are released in _technical preview_. This means that the feature can already be used but is not yet final or fully tested.  If you want to use such a feature, we recommend trying out the feature extensively in a test environment before using it in production. 
 
-Since features in technical preview are very new, we don't guarantee backwards compatibility during the time a feature is in technical preview. Such a feature may change based on feedback from users or from our internal usability testing. The documentation should contain all the necessary information to get started using it but it may not be complete and is also subject to change.
+Since features in a technical preview are very new, we don't guarantee backward compatibility during the time a feature is in technical preview. Such a feature may change based on feedback from users or our internal usability testing. The documentation should contain all the necessary information to get started using it but it may not be complete and is also subject to change.
 
-Features released in technical preview usually become standard full features in a subsequent quarterly version.
+Features released in technical preview usually become fully implemented in some future version but they might also disappear altogether.
```

Comment:
Might be better to state: "but there is the possibility that they might be removed entirely"